pages/ko/lb3/Installation.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 설치하기
+lang: ko
+keywords: LoopBack
+tags: [installation]
+sidebar: lb3_sidebar
+permalink: /doc/ko/lb3/Installation.html
+summary: Install the LoopBack CLI tool to create and develop LoopBack 3.0 apps.
+---
```

Comment:
Please add `layout: translation` so the page uses the proper layout, as described in http://loopback.io/doc/en/contrib/Translating_articles.html#procedure.

Also, for a partially-translated page such as this please add `trans_complete: false` to the frontmatter, as explained in http://loopback.io/doc/en/contrib/Translating_articles.html#front-matter-example.